app/views/system_admin/pending_trns/index.html.erb

```diff
@@ -8,43 +14,29 @@
-      <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-          <th scope="col" class="govuk-table__header">First names</th>
-          <th scope="col" class="govuk-table__header">Last name</th>
-          <th scope="col" class="govuk-table__header">Days waiting</th>
-          <th scope="col" class="govuk-table__header" colspan="3"></th>
-        </tr>
-      </thead>
-      <tbody class="govuk-table__body">
-        <% @trainees.each do |trainee| %>
-          <tr class="govuk-table__row">
-            <td class="govuk-table__cell"><%= trainee.first_names %></td>
-            <td class="govuk-table__cell"><%= trainee.last_name %></td>
-            <td class="govuk-table__cell">
-              <%= (Date.current - trainee.dqt_trn_request.created_at.to_date).to_i if trainee.dqt_trn_request.present? %>
-            </td>
-            <td class="govuk-table__cell">
-              <%= govuk_link_to "View", trainee_personal_details_path(trainee), class: "govuk-!-margin-0 govuk-button" %>
-            </td>
-            <td class="govuk-table__cell">
-              <%= govuk_button_to "Check for TRN",
-                                  pending_trns_retrieve_trn_path(trainee),
-                                  method: :put,
-                                  class: "govuk-!-margin-0 govuk-button--secondary" %>
-            </td>
-            <td class="govuk-table__cell">
-              <%= govuk_button_to "Re-submit for TRN",
-                                  pending_trns_request_trn_path(trainee),
-                                  method: :put,
-                                  class: "govuk-!-margin-0 govuk-button--secondary" %>
-            </td>
-          </tr>
-        <% end %>
-      </tbody>
-    </table>
+    <%= govuk_accordion do |accordion|
+      @trainees.each do |trainee|
+        accordion.section(heading_text: "#{trainee.first_names} #{trainee.last_name}") do
+          tag.div(class: "govuk-body") do
+            concat(
+              tag.p((Date.current - trainee.dqt_trn_request.created_at.to_date).to_i)
+            ) if trainee.dqt_trn_request.present?
+
+            concat(
+              govuk_details(summary_text: "Request Details") { concat(trainee.dqt_trn_request.response) }
+            ) if trainee.dqt_trn_request.present?
+
+            concat(
+              tag.div(class: "system-admin-actions") do
+                concat(govuk_link_to "View trainee", trainee_personal_details_path(trainee), class: "govuk-button govuk-!-margin-bottom-0")
+                concat(govuk_button_to "Check for TRN", pending_trns_retrieve_trn_path(trainee), method: :put, class: "govuk-!-margin-0 govuk-button--secondary")
+                concat(govuk_button_to "Re-submit for TRN", pending_trns_request_trn_path(trainee), method: :put, class: "govuk-!-margin-0 govuk-button--secondary")
+              end
+            )
+          end
+        end
```

Comment:
Probably best to rewrite it to 

```ruby
    <%= govuk_accordion do |accordion|
      @trainees.each do |trainee|
        accordion.section(heading_text: "#{trainee.first_names} #{trainee.last_name}") do  %>
        
        <div class="govuk-body">
          <%= tag.p((Date.current - trainee.dqt_trn_request.created_at.to_date).to_i) if trainee.dqt_trn_request.present? %>
          <%= govuk_details(summary_text: "Request Details") do %>
            <%= trainee.dqt_trn_request.response %>
          <% end if trainee.dqt_trn_request.present? %>
          <div class="system-admin-actions">

            <%= govuk_button_link_to("View trainee", trainee_personal_details_path(trainee), class: "govuk-!-margin-bottom-0")     %>
            <%= govuk_button_to("Check for TRN", pending_trns_retrieve_trn_path(trainee), method: :put, class: "govuk-!-margin-0", secondary: true)%>
            <%= govuk_button_to("Re-submit for TRN", pending_trns_request_trn_path(trainee), method: :put, class: "govuk-!-margin-0", secondary: true)%>
          </div>
        </div>
        <% end
      end
    end %>
```

Finding it quite hard with those `concat`

- replace `#{trainee.first_names} #{trainee.last_name}` with `#{trainee.short_name}`
- instead of `govuk_link_to` use `govuk_button_link_to` so the class can be drop `govuk-button`
- use `secondary: true` so the class can be drop `govuk-button--secondary`

`govuk-!-margin-0` probably not needed for each button maybe fix it in the parent.

Reply:
Yeah not sure why I didn't do that before 😅. I think we may end up changing the markup completely though based on Andrew's suggestion